backend/typescript/rest/entityRoutes.ts

```diff
@@ -0,0 +1,91 @@
+import express, { Router } from 'express';
+import { Entity } from "../models/entity.mgmodel"
```

Comment:
This should not be needed once the service layer uses `EntityRequestDTO`. We don't want the controller directly using the model.

Reply:
This import is not needed anymore